.travis.yml

```diff
@@ -14,16 +16,20 @@ install:
   - cd ../foreman
   - echo "gemspec :path => '../${FOREMAN_PLUGIN_NAME}'" > bundler.d/${FOREMAN_PLUGIN_NAME}.local.rb
   - ln -s settings.yaml.test config/settings.yaml
-  - ln -s database.yml.example config/database.yml
-  - bundle install --jobs=3 --retry=3 --without journald development postgresql mysql2 console journald
+  - bundle install --jobs=3 --retry=3 --without journald development mysql2 console journald
   - npm install
-  - bundle exec rake webpack:compile
-  - bundle exec rake db:migrate RAILS_ENV=test
 script:
+  - echo "export DATABASE_URL=postgresql://localhost/travis_ci_test" > test_env.sh
```

Comment:
Why not exporting the variable directly?

Reply:
done. :)